python/nav/web/webfront/views.py

```diff
@@ -83,21 +88,43 @@ def index(request, did=None):
         'title': 'NAV - {}'.format(dashboard.name),
     }
 
-    if dashboards.count() > 1:
-        dashboard_ids = [d.pk for d in dashboards]
-        current_index = dashboard_ids.index(dashboard.pk)
-        previous_index = current_index - 1
-        next_index = current_index + 1
-        if current_index == len(dashboard_ids) - 1:
-            next_index = 0
-        context.update(
-            {
-                'previous_dashboard': dashboards.get(pk=dashboard_ids[previous_index]),
-                'next_dashboard': dashboards.get(pk=dashboard_ids[next_index]),
-            }
+    return render(request, 'webfront/index.html', context)
+
+
+@require_POST
+def toggle_dashboard_shared(request, did):
+    """Toggle shared status for this dashboard"""
+    dashboard = get_object_or_404(AccountDashboard, pk=did, account=request.account)
+
+    is_shared_param = request.POST.get('is_shared')
+    is_shared = is_shared_param in ('True', 'on')
```

Comment:
Why can this be both `True` and `on`?

Reply:
The `input[type="checkbox"]` element sets its value to `on` when selected, but sends no parameters when unchecked. I added a hidden input for the unchecked case with `value="false"` to make sure the value is always sent in requests. In the tests I wanted to use `True` and `False` instead of `on` and `off`, so I had to add `True` as a valid truthy value. Maybe I should have used `on`/`off` instead to match the html implementation, e.g.
```python
self._post_toggle_shared(client, dashboard.id, 'off')
...
self._post_toggle_shared(client, dashboard.id, 'on')
```

Reply:
Yes, I think it would be good to be consistent (and for clarity you can mention in the test that it is `on` if the checkbox is ticked

Reply:
I added two fixups, where the `_post_toggle_shared/3` method takes care of setting the proper `'on'` and `'off'` values.